### ISSUE TITLE
no error/dialog upon attempting to send a transaction via a different account

### ISSUE THREAD

**Issue by wighawag** [MetaMask/hello#21]
There is no tx or error dialog nor any callback called when attempting to send a transaction via a different account that the one selected in metamask ui.

scenario :
user launch the dapp, switch to another account while the dapp is running,
perform a tx via the dapp (the dapp did not update yet to the new selected account) : metamask do not show any error nor callback with an error.

**Comment by kumavis** [comment]
Related to #20 
Fixed by #408

```
[Error: Error: Unknown address - unable to sign transaction for this address. at chrome-extension…, undefined]
Error: Error: Unknown address - unable to sign transaction for this address. at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:101705:35 at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:101726:5 at HookedWalletSubprovider.getAccounts (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:1711:11) at HookedWalletSubprovider.validateSender (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:101723:8) at HookedWalletSubprovider.validateTransaction (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:101703:8) at fn (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:4742:34) at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:5209:16 at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:4162:37 at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:4702:43 at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:4163:37 at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:101705:35 at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:101726:5 at HookedWalletSubprovider.getAccounts (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:1711:11) at HookedWalletSubprovider.validateSender (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:101723:8) at HookedWalletSubprovider.validateTransaction (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:101703:8) at fn (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:4742:34) at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:5209:16 at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:4162:37 at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:4702:43 at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/background.js:4163:37 at Object.InvalidResponse (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/inpage.js:11826:28111) at chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/inpage.js:11827:25427 at StreamProvider._onResponse (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/inpage.js:11804:3) at StreamProvider._write (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/inpage.js:11812:8) at doWrite (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/inpage.js:7963:64) at writeOrBuffer (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/inpage.js:7952:5) at StreamProvider.Writable.write (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/inpage.js:7890:11) at DestroyableTransform.ondata (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/inpage.js:9144:20) at DestroyableTransform.EventEmitter.emit (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/inpage.js:5380:17) at readableAddChunk (chrome-extension://pfejnnoghgohcpmiahgplcflfpbfeokf/scripts/inpage.js:8811:18)
```

**Comment by shikhars371** [comment]
its not yet fixed :(
